config/webpack.config.dev.js

```diff
@@ -141,13 +141,17 @@ module.exports = {
           // Process JS with Babel.
           {
             test: /\.(js|jsx|mjs)$/,
-            include: paths.appSrc,
+            include: [paths.appSrc, `${paths.appNodeModules}/@hashgraph`],
             loader: require.resolve('babel-loader'),
             options: {
               // This is a feature of `babel-loader` for webpack (not Babel itself).
               // It enables caching results in ./node_modules/.cache/babel-loader/
               // directory for faster rebuilds.
               cacheDirectory: true,
+              presets: [
```

Comment:
what does this do?

Reply:
Specify what babel should use while transpiling the files. The preset-env allows transpiling of newer version of js (ES2019), the react preset allows transpiling of react. Originally, the react preset was stated in the package.json, but I don't know why it does not work when I specify the `preset-env` in the package.json, so I just moved both in the config file themselves.